pkg/controller/clusterautoscaler/clusterautoscaler_controller.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 const (
-	controllerName      = "cluster-autoscaler-controller"
+	controllerName      = "cluster_autoscaler_controller"
```

Comment:
I know why you're doing this, but I really wish it was fixed in controller-runtime...